pkg/daemon/ceph/client/crush.go

```diff
@@ -100,6 +100,8 @@ func GetCrushMap(context *clusterd.Context, clusterName string) (CrushMap, error
 // FindOSDInCrushMap finds an OSD in the CRUSH map
 func FindOSDInCrushMap(context *clusterd.Context, clusterName string, osdID int) (*CrushFindResult, error) {
 	args := []string{"osd", "find", strconv.Itoa(osdID)}
+	cmd := NewCephCommand(context, clusterName, args)
+	cmd.Debug = true
```

Comment:
Hum it looks like `NewCephCommand` verbosity is controlled by `IsDebugLevel()` so I'm not sure why this is needed.

Reply:
It is initialized based on IsDebugLevel(), but line 104 overrides that default to force it to be debug level.

Reply:
So then if the Operator runs in INFO log we shouldn't see verbose logs, right? Why not sticking with `IsDebugLevel()`? I feel like I'm missing something...

Reply:
Based on our discussion, can we close this since it's not needed?